lib/galaxy/managers/users.py

```diff
@@ -609,7 +609,7 @@ def send_reset_email(self, trans, payload, **kwd):
                     log.debug(body)
                     return f"Failed to submit email. Please contact the administrator: {util.unicodify(e)}"
             else:
-                return "Failed to produce password reset token. User not found."
+                return None
```

Comment:
We no longer need the `else` clause: `None` will be returned by the outer conditional. To simplify, you could just have `return None` as the last line in the method's body, outside the conditionals' scope.

Reply:
Or just drop the `else:` clause completely, as returning None is the default.